scripts/reinforcement_learning/rsl_rl/cli_args.py

```diff
@@ -38,6 +39,30 @@ def add_rsl_rl_args(parser: argparse.ArgumentParser):
         "--log_project_name", type=str, default=None, help="Name of the logging project when using wandb or neptune."
     )
 
+    arg_group.add_argument(
+        "--wandb_run_id",
+        type=str,
+        default=None,
+        help=(
+            "Run ID for Weights & Biases (wandb) to load a specific run. If not provided, will not load the run from"
+            " wandb."
+        ),
+    )
+    arg_group.add_argument(
+        "--wandb_checkpoint_iteration",
+        type=str,
+        default=None,
+        help="Select which wandb checkpoint iteration to load. If not provided, the latest checkpoint will be used.",
+    )
+    arg_group.add_argument(
+        "--wandb_username",
+        type=str,
+        default=None,
```

Comment:
Would this make sense?

```suggestion
        default=os.environ["WANDB_USERNAME"],
```